ui/page/governance/proposal_details_page.go

```diff
@@ -579,3 +586,59 @@ func (pg *ProposalDetails) Layout(gtx C) D {
 	}
 	return components.UniformPadding(gtx, body)
 }
+
+func (pg *ProposalDetails) layoutMobile(gtx layout.Context) layout.Dimensions {
+	proposal := pg.proposal
+	_, ok := pg.proposalItems[proposal.Token]
+	if !ok && !pg.loadingDescription {
+		pg.loadingDescription = true
+		go func() {
+			var proposalDescription string
+			if proposal.IndexFile != "" && proposal.IndexFileVersion == proposal.Version {
+				proposalDescription = proposal.IndexFile
+			} else {
+				var err error
+				proposalDescription, err = pg.WL.MultiWallet.Politeia.FetchProposalDescription(proposal.Token)
+				if err != nil {
+					log.Errorf("Error loading proposal description: %v", err)
+					time.Sleep(7 * time.Second)
+					pg.loadingDescription = false
+					return
```

Comment:
Same review above applies here.

Reply:
we should leave as is, further details here https://github.com/planetdecred/godcr/pull/971#discussion_r904149266